tests/TestCollectorInit.py

```diff
@@ -5,20 +5,20 @@
 from unittest import TestCase
 from unittest.mock import call, patch, MagicMock
 import importlib
-import collectors.VMStatsCollector
+import collectors.HostSystemStatsCollector
 from exporter import initialize_collector_by_name
 
 
 
 
 class TestCollectorInitialization(TestCase):
     os.environ.setdefault('TARGET', 'testhost.test')
-    collectors.VMStatsCollector.BaseCollector.get_target_tokens = MagicMock(return_value={'testhost.test': '2ed214d52'})
+    collectors.HostSystemStatsCollector.BaseCollector.get_target_tokens = MagicMock(return_value={'testhost.test': '2ed214d52'})
```

Comment:
Why is this changing?

Reply:
Just for testing purposes. Changed it back.